src/components/EditorsColumn.jsx

```diff
@@ -13,58 +17,99 @@ function allErrorsFor(language, errors, runtimeErrors) {
+  return editorRefs.map(ref =>
+    ref ? ReactDOM.findDOMNode(ref).offsetHeight : null);
+}
+
+export default class EditorsColumn extends React.Component {
+  constructor(props) {
+    super(props);
+    bindAll(this, '_storeEditorRef', '_handleEditorDividerDrag');
+  }
+
+  _storeEditorRef(index, editor) {
+    this.editorRefs[index] = editor;
+  }
+
+  _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
+    const {onUpdateFlex} = this.props;
+    const heights = extractNodeHeights(this.editorRefs);
+    onUpdateFlex({deltaY, heights, index, lastY, y});
+  }
 
-    editors.push(
-      <EditorContainer
-        key={language}
-        language={language}
-        source={currentProject.sources[language]}
-        onMinimize={
-          partial(onComponentMinimize, `editor.${language}`)
-        }
-      >
-        <Editor
-          errors={allErrorsFor(language, errors, runtimeErrors)}
+  render() {
+    const {
+      currentProject,
+      editorsFlex,
+      errors,
+      onComponentMinimize,
+      onEditorInput,
+      onRequestedLineFocused,
+      runtimeErrors,
+      ui,
+    } = this.props;
+
+    const editors = [];
+    this.editorRefs = [];
+    const languages = ['html', 'css', 'javascript'].filter(language =>
+      !includes(ui.minimizedComponents, `editor.${language}`),
+    );
+    languages.forEach((language, index) => {
+      editors.push(
+        <EditorContainer
           key={language}
           language={language}
-          percentageOfHeight={1 / editors.length}
-          projectKey={currentProject.projectKey}
-          requestedFocusedLine={ui.editors.requestedFocusedLine}
           source={currentProject.sources[language]}
-          onInput={partial(onEditorInput, language)}
-          onRequestedLineFocused={onRequestedLineFocused}
-        />
-      </EditorContainer>,
-    );
-  });
+          style={{flex: editorsFlex[index], minHeight: EDITOR_MIN_HEIGHT}}
+          onMinimize={
+            partial(onComponentMinimize, `editor.${language}`)
+          }
+        >
+          <Editor
+            errors={allErrorsFor(language, errors, runtimeErrors)}
+            key={language}
+            language={language}
+            percentageOfHeight={1 / languages.length}
+            projectKey={currentProject.projectKey}
+            ref={partial(this._storeEditorRef, index)}
+            requestedFocusedLine={ui.editors.requestedFocusedLine}
+            source={currentProject.sources[language]}
+            onInput={partial(onEditorInput, language)}
+            onRequestedLineFocused={onRequestedLineFocused}
+          />
+          {(index < languages.length - 1) &&
+            <DraggableCore
+              axis="y"
+              bounds="parent"
+              key={`divider:${language}`}
+              onDrag={partial(this._handleEditorDividerDrag, index)}
+            >
+              <div
+                className="editors__divider"
+                style={{height: DIVIDER_HEIGHT}}
```

Comment:
Seems like we could maybe even use `getComputedStyle` to do the same thing for the `EDITOR_MIN_HEIGHT`? It would be really slick if those values were totally controlled by the stylesheet : )